x/concentrated-liquidity/client/query_proto_wrap.go

```diff
@@ -143,19 +148,22 @@ func (q Querier) LiquidityNetInDirection(ctx sdk.Context, req clquery.LiquidityN
 		ctx,
 		req.PoolId,
 		req.TokenIn,
-		startTick,
+		sdk.NewInt(startTick),
 		boundTick,
 	)
 	if err != nil {
 		return nil, err
 	}
 
-	pool, err := q.Keeper.GetConcentratedPoolById(ctx, req.PoolId)
-	if err != nil {
-		return nil, err
+	startTickLiquidity := sdk.ZeroDec()
+	for i := range liquidityDepths {
+		if liquidityDepths[i].TickIndex == startTick {
+			startTickLiquidity = liquidityDepths[i].LiquidityNet
+			break
+		}
 	}
```

Comment:
This doesn't look right to me. We should be grabbing liquidity from the current tick in pool model and adding each liquidity net as we iterate until we hit the start tick.

I think we should also cover this change with tests as this risks breaking the core quotes path for the frontend.

As discussed earlier let's leave out this issue until we address all remaining launch blockers. Thank you for looking into it

Reply:
Is this more so what you were thinking? I am going to switch over to the next item but am just pushing what I have so far since I think it is close

https://github.com/osmosis-labs/osmosis/pull/5466/commits/a8512ee3b14c4a41066b043cc033a9b656d53be4